magento/2/Magento-CE-2.4.5.tar.gz.md5

```diff
@@ -1 +1 @@
-8d1deeb4ba594861840a9dc1c45d8ba8
+a1996e49ea40fbe84822fd48f4bf962f
```

Comment:
Curious if there should be Magento related files in this PR

Reply:
I honestly have no idea where those came from.. They were definitely not touched by me.